sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,196 @@ def __repr__(self):
+    .. [Rogner2017] I. Rogner, "Friction modelling for robotic applications with planar motion",
+            Chalmers University of Technology, Department of Electrical Engineering, 2017.
+
+    """
+
+    def __init__(self, mu_k, f_n, pathway, *, v_s=None, sigma=0, mu_s=None):
+        self.mu_k = mu_k if mu_k is not None else 1
+        self.mu_s = mu_s if mu_s is not None else self.mu_k
+        self.f_n = f_n
+        self.sigma = sigma if sigma is not None else 0
+        self.v_s = v_s if v_s is not None or v_s == 0 else 0.01
+        self.pathway = pathway
+
+    @property
+    def mu_k(self):
+        """The coefficient of kinetic friction."""
+        return self._mu_k
+
+    @mu_k.setter
+    def mu_k(self, mu_k):
+        if hasattr(self, '_mu_k'):
+            msg = (
+                f'Can\'t set attribute `mu_k` to '
+                f'{repr(mu_k)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_k = sympify(mu_k, strict=True)
+
+    @property
+    def mu_s(self):
+        """The coefficient of static friction."""
+        return self._mu_s
+
+    @mu_s.setter
+    def mu_s(self, mu_s):
+        if hasattr(self, '_mu_s'):
+            msg = (
+                f'Can\'t set attribute `mu_s` to '
+                f'{repr(mu_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._mu_s = sympify(mu_s, strict=True)
+
+    @property
+    def f_n(self):
+        """The normal force between the surfaces."""
+        return self._f_n
+
+    @f_n.setter
+    def f_n(self, f_n):
+        if hasattr(self, '_f_n'):
+            msg = (
+                f'Can\'t set attribute `f_n` to '
+                f'{repr(f_n)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._f_n = sympify(f_n, strict=True)
+
+    @property
+    def sigma(self):
+        """The viscous friction coefficient."""
+        return self._sigma
+
+    @sigma.setter
+    def sigma(self, sigma):
+        if hasattr(self, '_sigma'):
+            msg = (
+                f'Can\'t set attribute `sigma` to '
+                f'{repr(sigma)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._sigma = sympify(sigma, strict=True)
+
+    @property
+    def v_s(self):
+        """The Stribeck friction coefficient."""
+        return self._v_s
+
+    @v_s.setter
+    def v_s(self, v_s):
+        if hasattr(self, '_v_s'):
+            msg = (
+                f'Can\'t set attribute `v_s` to '
+                f'{repr(v_s)} as it is immutable.'
+            )
+            raise AttributeError(msg)
+        self._v_s = sympify(v_s, strict=True)
+
+    @property
+    def force(self):
+        v = self.pathway.extension_velocity
+        f_c = self.mu_k * self.f_n
+        f_max = self.mu_s * self.f_n
+        stribeck_term = (f_max - f_c) * exp(-(v / self.v_s)**2) if self.v_s != 0 else 0
```

Comment:
This isn't the correct way to check if a symbol is zero. You need to use the `is_zero` attribute.

Reply:
I think it is better to make the optional parameters default to `None` in the function signature, that way you can check if None or not to include these force terms. Checking whether something is zero is more unreliable.